band/band_ru864_869.go

```diff
@@ -0,0 +1,101 @@
+package band
+
+import "time"
+
+func newRU864Band(repeaterCompatible bool) (Band, error) {
+	var maxPayloadSize []MaxPayloadSize
+
+	if repeaterCompatible {
+		maxPayloadSize = []MaxPayloadSize{
+			{M: 59, N: 51},
+			{M: 59, N: 51},
+			{M: 59, N: 51},
+			{M: 123, N: 115},
+			{M: 230, N: 222},
+			{M: 230, N: 222},
+			{M: 230, N: 222},
+			{M: 230, N: 222},
+		}
+	} else {
+		maxPayloadSize = []MaxPayloadSize{
+			{M: 59, N: 51},
+			{M: 59, N: 51},
+			{M: 59, N: 51},
+			{M: 123, N: 115},
+			{M: 250, N: 242},
+			{M: 250, N: 242},
+			{M: 250, N: 242},
+			{M: 250, N: 242},
+		}
+	}
+
+	return Band{
```

Comment:
Could you also implement `getPingSlotFrequencyFunc`? This is needed for Class-B support. For an example see: https://github.com/brocaar/lorawan/blob/master/band/band_eu863_870.go#L107.